test/hotspot/jtreg/testlibrary/ctw/Makefile

```diff
@@ -78,22 +81,32 @@ $(DST_DIR)/ctw.sh: $(DST_DIR)
 	echo '$${JAVA_HOME}/bin/java $${JAVA_OPTIONS} $(EXPORTS) -XX:-UseCounterDecay -Xbatch "-XX:CompileCommand=exclude,java/lang/invoke/MethodHandle.*" -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI -Xbootclasspath/a:wb.jar -jar ctw.jar $$@' > $@
 	chmod a+x $@
 
+$(DST_DIR)/ctwrunner.sh: $(DST_DIR)
+	echo '$${JAVA_HOME}/bin/java $${JAVA_OPTIONS} -Dtest.jdk=$${JAVA_HOME} -jar ctwrunner.jar $$@' > $@
+	chmod a+x $@
+
 $(DST_DIR)/ctw.jar: filelist $(DST_DIR)/wb.jar
 	@mkdir -p $(OUTPUT_DIR)
 	$(JAVAC) $(EXPORTS) -sourcepath $(SRC_DIR) -d $(OUTPUT_DIR) -cp $(DST_DIR)/wb.jar @filelist
-	$(JAR) --create --file=$@ --main-class $(MAIN_CLASS) -C $(OUTPUT_DIR) .
+	$(JAR) --create --file=$@ --main-class $(CTW_MAIN_CLASS) -C $(OUTPUT_DIR) .
+	@rm -rf $(OUTPUT_DIR)
+
+$(DST_DIR)/ctwrunner.jar: filelist $(DST_DIR)/wb.jar
```

Comment:
Theoretically, we can avoid from generating ctwrunner.jar. The classes are the same as 'ctw.jar' and the only different part is the entry point. 

to launch CTWRunner, you can do thing like this in ctwruner.sh
```
echo '$${JAVA_HOME}/bin/java $${JAVA_OPTIONS} -Dtest.jdk=$${JAVA_HOME} -cp ctw.jar $CTWRUNNER_MAIN_CLASS  $$@' > $@
```

it's up to you. if we generate ctwrunner.jar, we will have a "standalone" jar.

Reply:
I removed ctwrunner.jar in newest commit.

I also realized I was javac'ing twice. We avoid this entirely by removing the extra jar.